assets/sass/atoms/_overlays.scss

```diff
@@ -11,4 +11,24 @@
     right: 0px;
     background: rgba(255, 255, 255, 0.8);
   }
+  .fullscreen{
+    z-index: 100000;
+    position: fixed;
+    top: 0px;
+    left: 0px;
+    width: 100%;
+    height: 100%;
```

Comment:
seems dangerous to me... Even if it's a kind of “hack”, use `bottom: 0px;`, it's saffer. We can also use `height: 100vh;` as cleaner way to do it (Switzerland support -> 95.14%) :wink: